src/Models/Note.php

```diff
@@ -47,6 +58,23 @@ public function notable()
         return $this->morphTo();
     }
 
+    public function getNotableNameAttribute(): ?string
+    {
+        return $this->notable->name ?? null;
+    }
+
+    public function getNotableLinkAttribute(): ?string
+    {
+        if (! $this->notable->novaResourceClass) {
+            return null;
+        }
+
+        return route('nova.pages.detail', [
+            'resource' => app()->make($this->notable->novaResourceClass)::uriKey(),
+            'resourceId' => $this->notable->id,
+        ]);
```

Comment:
In order to create a proper link to a Laravel Nova resource this code relies on the notable model to have a novaResourceClass property, otherwise the link cannot be created